### ISSUE TITLE
image format for pretrained models

### ISSUE THREAD

**Issue by guillaumevr** [facebookresearch/maskrcnn-benchmark#410]
## ❓ Questions and Help
Hello,
I'd like to use the faster rcnn R50 pretrained model. What format of image should I provide to resnet? (consider that I'm not using your transform function)
Is it RGB? BGR? [0,255]? [0,1]? Should I normalize my data?
Is it different for pretrained models coming from 
e2e_faster_rcnn_R_50_FPN_1x_caffe2.yaml
and from
e2e_faster_rcnn_R_50_FPN_1x.yaml
Thank you

**Comment by fmassa** [comment]
Hi,

Check the webcam demo, there is some useful information there:
https://github.com/facebookresearch/maskrcnn-benchmark/blob/d3fed42afe59910dd650308c426640d183d044b5/demo/predictor.py#L138-L145

Also, the pre-trained models are expecting BGR 0-255 images (as output by OpenCV).

I'd recommend just taking the COCODemo from https://github.com/facebookresearch/maskrcnn-benchmark/blob/d3fed42afe59910dd650308c426640d183d044b5/demo/predictor.py#L14
and adapting it for your needs to begin with.

Also, the `e2e_faster_rcnn_R_50_FPN_1x.yaml` config is not pre-trained for detection, and you'd need to manually change the `MODEL.WEIGHT` path to the pre-trained models in https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/MODEL_ZOO.md#end-to-end-faster-and-mask-r-cnn-baselines

But I'd recommend just using the `caffe2` configs, as they give the same results (and we now also have ResNeXt152 support from Caffe2 model)

**Comment by guillaumevr** [comment]
Thank you very much
